File: Programmers/Inho/Level01/이상한 문자 만들기.cs
using System;

public class Solution
{
    public string solution(string s)
    {
        string answer = "";
        char[] div = new char[s.Length];

        div = s.ToCharArray();
        int count = 1;
        for (int i = 0; i < s.Length; i++)
        {
            if (count % 2 == 1)
            { // 홀수 - 대문자
                div[i] = Char.ToUpper(div[i]);
            }
            else // 짝수 - 소문자
                div[i] = char.ToLower(div[i]);
            count++;
            if (div[i] == ' ')
                count = 1;
        }
        Console.WriteLine();
        foreach (char a in div)
        {
            answer += a.ToString();
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/이상한 문자 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string s) {
    string answer = "";
    int cnt = 0;
    for(int i = 0;i<s.size();i++){
        if(cnt%2==0){
            answer += toupper(s[i]);
        }
        else
            answer += tolower(s[i]);
        cnt++;
        if(s[i]==' ')
            cnt=0;
        
    }
    return answer;
}